mkdocs.yml

```diff
@@ -52,6 +52,8 @@ theme:
 
   icon:
     repo: fontawesome/brands/github # Custom icon for link to GitHub repository in header
+    edit: material/pencil # Custom icon for the edit source button
+    view: material/eye # Custom icon for the view source button
```

Comment:
Preview of our custom icons:

![custom-pencil-edit-buttons](https://user-images.githubusercontent.com/34918129/210277056-9ed5afbd-997a-4f3f-97be-85dfd4aff1dd.png)

Preview of the default icons:

![default-buttons](https://user-images.githubusercontent.com/34918129/210277092-648a3c34-6047-439d-a074-f0a337798b48.png)

Let me know what you like best. 😉